### PR TITLE
Remove incorrect reason when logging

### PR DESCRIPTION
**Pull request by VirrageS**
**What this PR does / why we need it**: We are using `.Reason(err)` which could not be set and even if it's set it's not relevant to this part of the code.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*: None

**Special notes for your reviewer**: None

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by maya-r** [comment]
/lgtm
/test all

Would be nicer to have the text changes in a different commit as they don't fit the commit message (I think we keep the commits separate even on merge), but since they're only changing comments, it's not a big deal.

File: pkg/virt-launcher/virtwrap/manager.go [hunk 1]
```diff
@@ -753,7 +753,7 @@ func (l *LibvirtDomainManager) generateConverterContext(vmi *v1.VirtualMachineIn
 		sev := kutil.IsSEVVMI(vmi)
 
 		if !l.efiEnvironment.Bootable(secureBoot, sev) {
-			log.Log.Reason(err).Errorf("EFI OVMF roms missing for booting in EFI mode with SecureBoot=%v, SEV=%v", secureBoot, sev)
+			log.Log.Errorf("EFI OVMF roms missing for booting in EFI mode with SecureBoot=%v, SEV=%v", secureBoot, sev)
```
Comment (maiqueb) [inline pkg/virt-launcher/virtwrap/manager.go:756]: I share @maya-r view on https://github.com/kubevirt/kubevirt/pull/7567#issuecomment-1098107281 . I would prefer if you limited this PR to this single line. 

The comment changes you're adding are opinionated at best. 

The one about the logging is also superfluous.

Comment (maiqueb) [reply pkg/virt-launcher/virtwrap/manager.go:756]: BTW - forgot to mention: good catch in this one.

Comment (VirrageS) [reply pkg/virt-launcher/virtwrap/manager.go:756]: I've reverted most of the changes. I've left the ones that were fixed by my linter (adding space between `//` and the comment) - but if you want I can remove that as well.

Comment (maiqueb) [reply pkg/virt-launcher/virtwrap/manager.go:756]: Out of curiosity, was it *your* linter (which) or the one used in the project ? 

Fell free to keep it, no problem - I won't insist.

**Review by maiqueb — APPROVED** [review]
Thanks, good catch on logging a "ghost" error reason.

File: pkg/virt-launcher/virtwrap/manager.go (hunk 1 above)
Comment (VirrageS) [reply pkg/virt-launcher/virtwrap/manager.go:756]: It's mine, `golangci-lint` to be exact which my IDE runs on save.

**Review by xpivarc — COMMENTED** [review]
/approve
Thanks!

**Pull request merged**